.neutrinorc.js

```diff
@@ -29,6 +29,7 @@ if (process.env.REACT_APP_LIB_MODE) {
         coverageReporters: ['text', 'html'],
         snapshotSerializers: ['jest-serializer-html'],
         setupFilesAfterEnv: ['<rootDir>/src/setupTests.js'],
+        globalSetup: './global-setup.js',
```

Comment:
I have added a global setup file for jest in order to set the timezone the UTC. Otherwise, the GitHub checks would have used UTC, whereas my local would have used CEST.